projects/packages/identity-crisis/src/scss/jetpack-idc.scss

```diff
@@ -1,4 +1,4 @@
-@import '../_inc/client/scss/variables/_colors.scss';
+@import './variables/_colors.scss';
```

Comment:
Do we really want to move this file at this point? Aren't we just adding the packages and duplicating some files? Moving this wouldn't break the plugin now?

Reply:
nope removal is in 19126